.pylintrc

```diff
@@ -243,13 +243,6 @@ max-line-length=100
 # Maximum number of lines in a module.
 max-module-lines=1000
 
-# List of optional constructs for which whitespace checking is disabled. `dict-
```

Comment:
removed as it seems this option has been removed from pylint: https://pylint.pycqa.org/en/v2.13.9/whatsnew/2.6.html - was getting a pylint error otherwise.